index.js

```diff
@@ -1944,11 +1979,16 @@
     });
   }
 
-  //  invalidArgumentsLength :: (String, Integer, Integer) -> Error
-  function invalidArgumentsLength(name, expectedLength, actualLength) {
-    return new TypeError(q(name) +
-                         ' requires ' + numArgs(expectedLength) + ';' +
-                         ' received ' + numArgs(actualLength));
+  //  tooManyArguments :: (TypeInfo, Integer) -> Error
```

Comment:
Great! Thank you, Irakli, for prompting me to explain these changes. Recording such things is important, but I will often neglect to do so unless someone asks a question.